requirements/github.txt

```diff
@@ -0,0 +1,4 @@
+# This requirement breaks both pip-tools and versioneye. The requirements are unchanged from master. It is possible to
+# temporarily change this to fs for updating pinned versions using pip-tools, then change it back after, but you will
+# have to hand edit the resulting requirements.txt to replace this url since it contains changes necessary for Python 3.
```

Comment:
Maybe reference [Cale's pip-tools PR](https://github.com/nvie/pip-tools/pull/405) here so it's easier to figure out when the comment is no longer relevant?